docs/static/releasenotes.asciidoc

```diff
@@ -65,6 +101,16 @@ No user-facing changes in Logstash core and plugins.
 ** `java.nio.file.NoSuchFileException` when finalizing the segment https://github.com/elastic/logstash/pull/15233[#15233]
 ** DLQ file using wrong sort order https://github.com/elastic/logstash/pull/15246[#15246]
 
+==== Notable plugin issues
```

Comment:
WDYT @roaksoax for this change?
Noting plugin issues and forwarding to the details would give us a flexibility to update the description when we bundle `output-email` plugin new version and unbundle the `input-imap`.

Reply:
Also noting that these notes  (Lines 104-113) were added to the 8.10.2 (not 8.10.3) release notes section.  Was that your intention?

Reply:
It is intentionally placed in 8.10.0 section. We came to know recently but plugins started having an issue after 8.10.0

Reply:
I'm interested to hear what @roaksoax says, but this info seems like it belongs in the 8.10.3 section as "Known  issues."  You can probably find an example if you search source.  When we have Known Issues, they are added as the first section 

We shouldn't expect users looking at the 8.10.3 release notes to know that they should go to the 8.10.0 release notes to see if we've added anything.